packages/shared/components/inputs/Pin.svelte

```diff
@@ -134,6 +134,10 @@
         }
     }
 
+    export function blur(): void {
```

Comment:
this also could be removed?

Reply:
I added this function to blur the sub inputs after the pin code has changed. Otherwise it should be possible to add a single digit at the end of the input you focused when submitting the new pin code. Like this:

<img width="320" alt="image" src="https://user-images.githubusercontent.com/6748220/191687688-223317ae-8794-4163-8729-bd957e4e7b24.png">